importer/importer_utils.py

```diff
@@ -468,15 +468,24 @@ def get_random_list_sample(some_list, amount):
     return random.sample(some_list, amount)
 
 
-def get_P31(q_number, site):
+def get_value_of_property(q_number, property_id, site):
     results = []
     item = pywikibot.ItemPage(site, q_number)
-    if item.exists() and item.claims.get("P31"):
-        for claim in item.claims.get("P31"):
-            results.append(claim.getTarget().getID())
+    if item.exists() and item.claims.get(property_id):
+        for claim in item.claims.get(property_id):
+            target = claim.getTarget()
+            if isinstance(target, str):
```

Comment:
I think it might be safer to use
```python
if isinstance(target, pywikibot.ItemPage):
    target = target.getID()
```
Since only `pywikibot.ItemPage` has `getID()`. The current approach would fail if e.g. the value was a WbTime.

(no need for the `if isinstance(target, str): target = target` since it doesn't change anything).